models/izhikevich.h

```diff
@@ -49,8 +49,8 @@ Implementation of the simple spiking neuron model introduced by Izhikevich
 
 .. math::
 
-   dV_m/dt &= 0.04 V_m^2 + 5 V_m + 140 - u + I
-   du/dt &= a (b V_m - u)
+   &\;dV_m/dt = 0.04 V_m^2 + 5 V_m + 140 - u + I\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\;\; \\
```

Comment:
Why all the small spaces; could you use something like \quad or align/&? Is the spacing even necessary?

Reply:
Hi, I added the spaces to center the formula, without them, it will look like this. Let me know if it's better
![izhikevich](https://github.com/nest/nest-simulator/assets/95964498/86c0c6c2-100e-4f09-8c85-c8d8cac2fe96)